src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -86,15 +87,18 @@ public static class UmbracoBuilderExtensions
 
             IHostingEnvironment tempHostingEnvironment = GetTemporaryHostingEnvironment(webHostEnvironment, config);
 
-            var loggingDir = tempHostingEnvironment.MapPathContentRoot(Cms.Core.Constants.SystemDirectories.LogFiles);
+            var loggingDir = tempHostingEnvironment.MapPathContentRoot(Constants.SystemDirectories.LogFiles);
             var loggingConfig = new LoggingConfiguration(loggingDir);
 
             services.AddLogger(tempHostingEnvironment, loggingConfig, config);
 
+            // TODO: This doesn't seem right? The HttpContextAccessor is normally added to the container
+            // with ASP.NET Core's own ext methods. Is there a chance we can end up with a different
+            // accessor registered and resolved?
```

Comment:
Seems like we are also doint this in `AddWebComponents` At least, we should not register it twice

Reply:
Yeah, that's gonna end up causing unwanted behavior IMO. I'll have a look

Reply:
I guess it won't cause problems because that's all based on AsyncLocal access, see https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http/src/HttpContextAccessor.cs ... but it will mean there's different HttpContextAccessor instances.  I'll just remove the 2nd one.